changelog.md

```diff
@@ -30,7 +30,7 @@
 * Added collection of `EndDeviceFunctionKind` to `EndDevice`
 * Added an unordered collection comparator.
 * Added the energized relationship for the current state of network between `Feeder` and `LvFeeder`.
-* Updated `NetworkConsumer.getEquipmentForContainers` to allow requesting normal, current or all equipments.
+* Updated `NetworkConsumer`'s `getEquipmentForContainers` and `getEquipmentContainers` to allow requesting normal, current or all equipments. 
```

Comment:
todo: Can you also do this for the loop requests. Easiest way to find where you need to do it is to remove the default value for the `networkState` parameter for `getEquipmentForContainers` to highlight them, then put it back in after for users of the SDK.

Reply:
added network state for loop